actors/market/tests/harness.rs

```diff
@@ -325,12 +327,27 @@ pub fn activate_deals_raw(
     provider: Address,
     current_epoch: ChainEpoch,
     deal_ids: &[DealID],
+    deals: &[DealProposal],
 ) -> Result<Option<IpldBlock>, ActorError> {
     rt.set_epoch(current_epoch);
     rt.set_caller(*MINER_ACTOR_CODE_ID, provider);
     rt.expect_validate_caller_type(vec![Type::Miner]);
 
-    let params = ActivateDealsParams { deal_ids: deal_ids.to_vec(), sector_expiry };
+    for deal in deals {
+        let piece = PieceInfo { cid: deal.piece_cid, size: deal.piece_size };
+        rt.expect_compute_unsealed_sector_cid(
+            RegisteredSealProof::StackedDRG8MiBV1,
+            vec![piece],
+            deal.piece_cid,
+            ExitCode::OK,
+        );
+    }
```

Comment:
This should be expecting a single compute_unsealed_sector_cid call with a vector of all the pieces at once.